src/supporting-data/keywords/components/entry/styles/keywords-graph.module.scss

```diff
@@ -0,0 +1,32 @@
+.graph {
+  position: relative;
+  width: 500px;
+  height: calc(50vh - 100px);
+}
+
+.container {
+  width: 500px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: calc(50vh - 100px);
+  position: absolute;
+
+  :global(.node) {
+    width: min-content;
+  }
+}
+
+.keyword-node {
```

Comment:
Would it be possible for short node labels to not wrap? Eg 2Fe-2S here:
<img width="106" alt="Screenshot 2022-06-16 at 08 36 42" src="https://user-images.githubusercontent.com/4357962/174017799-d0081224-bfe7-4bf3-b28e-d011168d305a.png">